tests/installation/partitioning_filesystem.pm

```diff
@@ -49,15 +49,15 @@ sub run {
     # select filesystem
     send_key 'home';
     send_key_until_needlematch("filesystem-$fs", 'down', 20, 3);
-    send_key 'ret' if check_var('DESKTOP', 'textmode');
+    send_key 'ret' if check_var('VIDEOMODE', 'text');
```

Comment:
In your verification run both settings are both matching for text mode and both are missing in case of gnome. So, what does it change/fix?

Reply:
ah, right, fixing these:
- https://openqa.suse.de/tests/1397278#step/partitioning_filesystem/9
- https://openqa.suse.de/tests/1399419#step/partitioning_filesystem/9

Already merge, but to state why.

Reply:
the instructions depends on the installation mode, which is controlled by VIDEOMODE=text